Objective-C/Tests/URLEndpointListenerTest.m

```diff
@@ -1108,6 +1108,7 @@ - (void) testMultipleReplicatorsOnReadOnlyListener {
 - (void) testReplicatorAndListenerOnSameDatabase {
     if (!self.keyChainAccessAllowed) return;
     
+    timeout = 5.0;
```

Comment:
Is this your intention to set the timeout to 5 second? Will this affect the other tests?

Reply:
https://github.com/couchbase/couchbase-lite-ios/blob/master/Objective-C/Tests/ReplicatorTest.m#L82

Its reset inside setup() method.

Reply:
this will be reset inside the Replicator>setup() method 
https://github.com/couchbase/couchbase-lite-ios/blob/master/Objective-C/Tests/ReplicatorTest.m#L82